### ISSUE TITLE
Download popup UI requires Redesign.

### ISSUE THREAD

**Issue by skyGtm** [TeamNewPipe/NewPipe#5508]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

Currently, the placement of OK button is in uncomfortable and weird position.
1. Weird, because when first time downloading using NewPipe, i didn't notice that button, because, most apps has OK button at the bottom of popup dialog.
2. Uncomfortable, because most mobile phone screen nowdays is longer and it is difficult to Press that OK button while using the device using single hand.

So I think it would be better to implement Bottom Sheet (I found a video demo here: https://www.youtube.com/watch?v=vw3BwdjqIIo)

#### Additional context

![Screenshot_2021-01-30-11-19-02](https://user-images.githubusercontent.com/50160188/106348516-e2b23380-62ee-11eb-9071-cdd6b8533117.png)

#### How will you/everyone benefit from this feature?

It improves the UX for every people with different device sizes.

**Comment by alienthief** [comment]
@skyGtm also the OK button is a bit out of context, Start would have been much clearer. could you add that to the description?

**Comment by alienthief** [comment]
@skyGtm I agree with you and also the OK button is a bit out of context, Start would have been much clearer. could you add that to the description?

**Comment by skyGtm** [comment]
@mhmdanas i think this issue should be labelled 'GUI', so that any potential contributor won't miss this issue. Thank you.
And if possible or suitable label available, please try to label my other issues too
